src/tree/registries/Azure/ACROAuthProvider.ts

```diff
@@ -105,6 +106,23 @@ export class ACROAuthProvider implements AuthenticationProvider {
         // Registry scopes, i.e. those passed to `getSession()`, are not valid for acquiring this
         // access token--instead, those only need to be passed to `getOAuthTokenFromRefreshToken()`
         const token = await subscription.credential.getToken([]);
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
         return token!.token;
     }
+

```

Comment:
just copied over the get refresh token logic from `getSession()` and made it a helper method since it's also used in `getLoginInformation()`